Add Pregnancy and Breasfeeding Therapeutic Areas

AlanGabbianelli opened the pull request:
This refers to https://www.gov.uk/drug-safety-update which is edited by
MHRA (Medicines and Healthcare products Regulatory Agency) in
https://specialist-publisher.publishing.service.gov.uk/drug-safety-updates

This is needed because they are launching a major new, lengthy project
on medicines and pregnant women, so will have a lot more content in this
area going forward, and wish to allow users to group existing content on
these topics closely together.

The request to add these two Therapeutic Areas came through Zendesk:
https://govuk.zendesk.com/agent/tickets/4099517

Related PRs:
- https://github.com/alphagov/search-api/pull/2113
- https://github.com/alphagov/govuk-content-schemas/pull/993

1pretz1 commented on lib/documents/schemas/drug_safety_updates.json:
```diff
@@ -29,6 +29,7 @@
       "filterable": true,
       "allowed_values": [
         {"label": "Anaesthesia and intensive care", "value": "anaesthesia-intensive-care"},
+        {"label": "Breasfeeding", "value": "breasfeeding"},
```
I'm assuming this should be "Breastfeeding" and "breastfeeding" respectively (missing t)

AlanGabbianelli replied:
Ah, I copied directly from the Zendesk ticket and hadn't noticed the typo, thank you! 🦅👁

1pretz1 reviewed (commented):
I don't know much about specialist-publisher but your change looks reasonable to me however I think the user's typo has made it's way from Zendesk to this PR.

Commit:
Add Pregnancy and Breasfeeding Therapeutic Areas

This refers to https://www.gov.uk/drug-safety-update which is edited by
MHRA (Medicines and Healthcare products Regulatory Agency) in
https://specialist-publisher.publishing.service.gov.uk/drug-safety-updates

This is needed because they are launching a major new, lengthy project
on medicines and pregnant women, so will have a lot more content in this
area going forward, and wish to allow users to group existing content on
these topics closely together.

The request to add these two Therapeutic Areas came through Zendesk:
https://govuk.zendesk.com/agent/tickets/4099517

Related PRs:
- https://github.com/alphagov/search-api/pull/2113
- https://github.com/alphagov/govuk-content-schemas/pull/993
